<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="image1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Image</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="image2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Manupilated Image</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="imageLine">
          <property name="placeholderText">
           <string>Select Bmp Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="leftButton">
          <property name="text">
           <string>Slider</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="imageButton">
          <property name="text">
           <string>Select Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="outNameLine">
          <property name="placeholderText">
           <string>Write Output Name (exp: grayimage)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Convert Grayscale</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Zoom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gaussian Mask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Draw Rect</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Draw Circle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Draw Ellipse</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Laplace Filter</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Histogram</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Histogram Equalization</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>KMeans</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Multi Dimensional KMeans</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Erosion</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dilation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Connected Component</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyButton">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
